sns/cm/repair/ut/xform.c

```diff
@@ -438,6 +438,87 @@ static void cp_multi_failures_post(char data, int cnt, int index)
 	m0_semaphore_down(&sem);
 }
 
+static void buf_initialize(struct m0_buf *buf, uint32_t size, uint32_t len)
+{
+	uint32_t j;
+	uint8_t	*arr;
+
+	for (j = 0; j < size; ++j) {
+		M0_ALLOC_ARR(arr, len);
+		M0_UT_ASSERT(arr != NULL);
+		m0_buf_init(&buf[j], arr, len);
+	}
+}
+
+static void buf_free(struct m0_buf *buf, uint32_t count)
```

Comment:
static void buf_free(struct m0_buf *bufs, uint32_t count)

Reply:
Good to use "count" here.
Maybe we can use the same name "count" in previous function.

Reply:
Done.